CONTRIBUTING.md

```diff
@@ -18,12 +18,26 @@ pip install -e ".[dev]"
 
 &nbsp;
 
-## Unit Tests and Recipe Tests
```

Comment:
Is there a note anywhere in here that says you need AWS access to see certain checkpoints?

Reply:
I mentioned it under the recipe tests section, do you think it should be more visible?

Reply:
Updated